functions/src/firestore-types/lobby.ts

```diff
@@ -51,6 +51,19 @@ export const GAME_STATE_DURATIONS: { [state in GameState]: number } = {
   END: 10,
 };
 
+/**
+ * whether to show timer or not
+ */
+export const DISPLAY_TIMERS: { [state in GameState]: boolean | null } = {
+  WAIT: null,
+  ROLE: false,
+  PROMPT: true,
+  CHAT: true,
+  VOTE: true,
+  RESULT: false,
+  END: false,
+};
```

Comment:
This file is for things in the database and DISPLAY_TIMERS seams to only be relivant to the client